eventstore/store.go

```diff
@@ -56,7 +56,7 @@ type Store struct {
 
 	lock       sync.RWMutex
 	models     map[reflect.Type]*Model
-	modelNames map[string]struct{}
+	modelNames map[string]*Model
```

Comment:
If we do this, it means a registered model can be retrieved by name. Is there any reason to keep the `models` field?

Reply:
Maybe not. At first I checked the model re-registration check with the Go Type, and thus using the `models` map. Now that identities are their name, isn't necessary.

Reply:
Ok, cool. I'll make that change.